spec/lib/tmuxinator/project_spec.rb

```diff
@@ -424,4 +424,33 @@
       end
     end
   end
+
+  describe "::load" do
+    let(:path) { File.expand_path("../../../fixtures/sample.yml", __FILE__) }
+    let(:options) { {} }
+
+    it "should raise if the project file doesn't parse" do
+      bad_yaml = <<-Y
+      name: "foo"
+        subkey:
+      Y
+      expect(File).to receive(:read).with(path){ bad_yaml }
+      expect{ self.described_class.load(path, options) }.to raise_error RuntimeError, %r{Failed.to.parse.config.file}
```

Comment:
Line is too long. [117/80]<br>Redundant `self` detected.<br>Space missing to the left of {.